WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -90,10 +101,14 @@ - (instancetype)initWithPost:(AbstractPost *)aPost shouldHideStatusBar:(BOOL)sho
     return self;
 }
 
+#pragma mark - UIViewController
+
 - (void)viewDidLoad
 {
     [super viewDidLoad];
 
+    [self setupReachability];
```

Comment:
Even if it wasn't strictly synchronous, but before the rest of the setup completes, it would be a bad thing.

I agree.